index-import.cpp

```diff
@@ -242,13 +225,38 @@ static bool cloneRecords(StringRef recordsDirectory,
 int main(int argc, char **argv) {
   cl::ParseCommandLineOptions(argc, argv);
 
-  auto remapper = Remapper::createFromCommandLine(PathRemaps);
+  Remapper remapper;
+  // Parse the the path remapping command line flags. This converts strings of
+  // "X=Y" into a (regex, string) pair. Another way of looking at it: each
+  // remap is equivalent to the s/pattern/replacement/ operator.
+  auto errors = 0;
+  for (const auto &remap : PathRemaps) {
+    auto divider = remap.find('=');
+    auto pattern = remap.substr(0, divider);
+    std::regex re;
+    try {
+      re = std::regex(pattern);
```

Comment:
The declaration and initialization can be combined here.

Reply:
done